install/wipe_ncn_disks_for_reinstallation.md

```diff
@@ -228,34 +262,83 @@ RAIDs, zeroing the disks, and then wiping the disks and RAIDs.
         7741d50966259410298bb4c3210e6665cdbd57a82e34e467d239f519ae3f17d4
         ```
 
-   1. Remove metal LVM.
+1. Remove etcd device **on master nodes ONLY**.
 
-        ```bash
-        ncn# vgremove -f -v --select 'vg_name=~metal*'
-        ```
+   1. This `dmsetup` comand wil determine whether an etcd volume is present.
 
-      > **NOTE:** Optionally, run the `pvs` command. If any drives are still listed, remove them with `pvremove`, but this is rarely needed. Also, if the above command fails or returns a warning about the filesystem being in use, ignore the error and proceed to the next step, as this will not inhibit the wipe process.
+      ```bash
+      ncn-m# dmsetup -ls 
+      ```
 
-1. Stop the RAIDs.
+      Expected output when the etcd volume is present
 
-    ```bash
-    ncn# for md in /dev/md/*; do mdadm -S -v $md || echo nope ; done
-    ```
+      ```bash
+      ETCDLVM (254:1)
```

Comment:
I don't think this will vary, but we should confirm it.  it could possibly be different depending on the hardware.  This is referring to the (254:1), not the ETCDLVM part of the output.   not a blocker.